resources/js/Pages/Frontend/Post.tsx

```
@@ -92,19 +126,19 @@ const Post = ({ post, status, board, votes }: PageProps<Props>) => {
+                className="h-7 w-7 rounded-full"
               />
             </div>
             <div className="flex-1">
-              <div className="text-sm font-semibold dark:text-gray-300 mb-3">
+              <div className="mb-3 text-sm font-semibold dark:text-gray-300">
                 {post.creator?.name}
               </div>
               <div
-                className="text-sm text-gray-800 dark:text-gray-300 mb-3"
+                className="mb-3 max-w-full overflow-hidden whitespace-normal break-words break-all text-sm text-gray-800 dark:text-gray-300"
                 dangerouslySetInnerHTML={{ __html: post.body }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)